packages/analytics-js-integrations/src/integrations/Amplitude/browser.js

```diff
@@ -85,14 +85,23 @@ class Amplitude {
 
     // rudderElement.message.context will always be present as part of identify event payload.
     const { traits } = rudderElement.message.context;
-    const { userId } = rudderElement.message;
-
+    const { userId, integrations } = rudderElement.message;
+    const amplitudeIntgConfig = getDestinationOptions(integrations);
+    const fieldsToUnset = amplitudeIntgConfig?.fieldsToUnset || undefined;
+    const amplitudeIdentify = new window.amplitude.Identify();
+    let sendIdentifyCall = false;
+    if (fieldsToUnset) {
+      sendIdentifyCall = true;
+      fieldsToUnset.forEach(fieldToUnset => {
+        amplitudeIdentify.unset(fieldToUnset);
```

Comment:
[Q] Can we collect all the fields to unset and call `amplitudeIdentify.unset` only once ? -- Just asking